pkg/exchange/okex/exchange.go

```diff
@@ -564,33 +567,31 @@ func (e *Exchange) QueryTrades(ctx context.Context, symbol string, options *type
 		return nil, ErrSymbolRequired
 	}
 
-	req := e.client.NewGetTransactionHistoryRequest().InstrumentID(toLocalSymbol(symbol))
-
 	limit := options.Limit
 	if limit > defaultQueryLimit || limit <= 0 {
 		log.Infof("limit is exceeded default limit %d or zero, got: %d, use default limit", defaultQueryLimit, limit)
 		limit = defaultQueryLimit
 	}
-	req.Limit(uint64(limit))
 
-	var newStartTime time.Time
+	timeNow := e.timeNowFunc()
```

Comment:
can be just `now` ?

Reply:
No, by using that `timeNowFunc()`, we can test the function.